packages/semantic-release-commit-rules/lib/tests/commit-analyzer.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const { expect} = require('chai');
+const getReleaseRules = require('../commit-analyzer');
+const config = require('./fixtures/configuration');
+
+describe('getReleaseRules exists', () => {
+  it('must be a function', () => {
+    expect(getReleaseRules).to.be.a('function');
+  })
+});
+
+describe('getReleaseRules generates valid releaseRules arrays', () => {
+  const releaseRules = getReleaseRules(config);
+
+  it('creates the correct number of rules', () => {
+    expect(releaseRules).to.have.a.lengthOf(6);
+  });
+
+  it('creates the correct type', () => {
+    expect(releaseRules[0].type).to.equal('feat');
```

Comment:
if you created an `expected` object that shows the expected output, which would be the releaseRules object. It would help understanding of what the expected output of the function would be.

```
"releaseRules": [
        {"type": "feat", "release": "minor"},
        {"type": "perf", "release": "patch"},
        {"type": "perf",...
```

Then you could reduce the individual tests to track, just doing a `to.deepEqual`

`expect(releaseRules).to.deepEqual(expected)`

Could reduce the tests needed and give an example of what the output should be.